doc/api/util.md

```diff
@@ -71,6 +71,25 @@ callbackFunction((err, ret) => {
 });
 ```
 
+## `util.colorText(format, text)`
+
+<!-- YAML
+added: v18.3.0
```

Comment:
This is out of date. The following change will let it be set automatically during the release process.

```suggestion
added: REPLACEME
```